src/components/buttons/ButtonsContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { View } from 'react-native'
+
+import { styled } from '../hoc/styled'
+import { MainButton, MainButtonType } from '../themed/MainButton'
+
+export interface ButtonContainerProps {
+  label: string
+  onPress: () => void | Promise<void>
+  disabled?: boolean
+}
+
+interface Props {
+  buttons: Map<MainButtonType, ButtonContainerProps>
+  layout: 'row' | 'column'
```

Comment:
Get rid of the map, and instead do `primary?: ButtonInfo; secondary?: ButtonInfo; escape?: ButtonInfo`.

Since we know the keys this component expects, we should not use a map, but a regular static object. Maps are for arbitrary key / value setups like loginId -> last login or something.